### DISCUSSION TITLE
Risk free rate

### DISCUSSION THREAD

**Discussion by tlurquin** [ranaroussi/quantstats#204] [General]
Hello, 

Thanks for the hard work on this super helpful package - I have a quick question about feeding a risk free rate into some of the method calls. Specifically, I would like to use the 3 month t-bill rate (a pandas.series with daily observations going back 40+ years). But feeding the series into: qs.reports.html(port_series, benchmark = mkt_series, rf= rf_series, ...) generates an error: ValueError: The truth value of a Series is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all()

Any and all insights/advice would be appreciated!
Thank you so much, 
Thomas

**Comment by AkaP88** [comment]
Hi!

It is actually expecting a float not a series. So if you're using 3M Libor (or other RF rate), you would need an annualized return of investment in it for the period of your backtest, i.e. computing a Total Return from your 3M rates over the period of backtesting and feeding the rf parameter with the CAGR of that series.

Hope it helps
